# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../util-linux/fix-nls-disabled.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- util-linux-2.13-pre7/include/nls.h.orig	2007-06-19 08:31:42.000000000 +0200
+++ util-linux-2.13-pre7/include/nls.h	2007-06-19 08:30:49.000000000 +0200
@@ -13,6 +13,7 @@
 #  define N_(String) (String)
 # endif
 #else
+# include <locale.h>
 # undef bindtextdomain
 # define bindtextdomain(Domain, Directory) /* empty */
 # undef textdomain
